common.cmake

```diff
@@ -41,6 +41,18 @@ function(nrfxlib_calculate_lib_path lib_path)
   set(${lib_path} "lib/${arch_soc_dir}/${float_dir}${short_wchar}" PARENT_SCOPE)
 endfunction()
 
+function(nrfxlib_check_softfp_path lib_name)
+  if (CONFIG_FP_SOFTABI)
+    set(lib_full_path ${${lib_name}})
+    string(REGEX REPLACE "softfp-float" "soft-float" soft_path ${lib_full_path})
```

Comment:
there is a minor risk that if someone has installed the workspace inside:
`/path/softfp-float/.../nrfxlib` then this will fail as all occurrences of `softfp-float` will be replaced.

Just as a general precaution, and because of the risk of such constructs might be copied elsewhere in the system, could you do the path replacing on the relative path you have, for example here:
https://github.com/nrfconnect/sdk-nrfxlib/blob/f25aa4ddee87cc525f66d9de3808426b2ef36afb/common.cmake#L41
or here:
https://github.com/nrfconnect/sdk-nrfxlib/blob/f25aa4ddee87cc525f66d9de3808426b2ef36afb/crypto/CMakeLists.txt#L18

before you make the path absolute ?